### ISSUE TITLE
Stylus Problem Still

### ISSUE THREAD

**Issue by dereklin** [Swiip/generator-gulp-angular#618]
I have reported this problem before.  And I just updated to version 0.12.0.
```
{
  "generator-gulp-angular": {
    "version": "0.12.0",
    "props": {
      "angularVersion": "~1.4.0",
      "angularModules": [
        {
          "key": "animate",
          "module": "ngAnimate"
        }
      ],
      "jQuery": {
        "key": "jquery2"
      },
      "resource": {
        "key": "none",
        "module": null
      },
      "router": {
        "key": "ui-router",
        "module": "ui.router"
      },
      "ui": {
        "key": "bootstrap",
        "module": null
      },
      "bootstrapComponents": {
        "key": "ui-bootstrap",
        "module": "ui.bootstrap"
      },
      "cssPreprocessor": {
        "key": "stylus",
        "extension": "styl"
      },
      "jsPreprocessor": {
        "key": "none",
        "extension": "js",
        "srcExtension": "js"
      },
      "htmlPreprocessor": {
        "key": "none",
        "extension": "html"
      },
      "foundationComponents": {
        "name": null,
        "version": null,
        "key": null,
        "module": null
      },
      "paths": {
        "src": "src",
        "dist": "dist",
        "e2e": "e2e",
        "tmp": ".tmp"
      }
    }
  }
}
```
When I run gulp, here is what I get:

```
c:\Temp\test1>gulp
[10:23:53] Using gulpfile c:\Temp\test1\gulpfile.js
[10:23:53] Starting 'clean'...
[10:23:53] Finished 'clean' after 52 ms
[10:23:53] Starting 'default'...
[10:23:53] Starting 'scripts'...
[10:23:53] Starting 'styles'...
[10:23:54] Starting 'partials'...
[10:23:54] Starting 'fonts'...
[10:23:54] Starting 'other'...
[10:23:54] Finished 'default' after 1.03 s
[10:23:54] gulp-inject 2 files into index.styl.
[10:23:55] [Stylus] Error in plugin 'gulp-stylus'
Message:
    c:\Temp\test1\src\app\index.styl:33:9
   29|  *  all your stylus files automatically
   30|  */
   31| // injector
   32| // endinjector
   33|
---------------^

failed to locate @import file src/app/components/malarkey/malarkey.styl

Details:
    lineno: 33
    column: 9
    filename: c:\Temp\test1\src\app\index.styl
    stylusStack:
[10:23:55] Finished 'styles' after 1.68 s
[10:23:55] all files 7.92 kB
[10:23:55] Finished 'scripts' after 2.03 s
[10:23:55] Starting 'inject'...
[10:23:55] Finished 'partials' after 1.33 s
[10:23:55] gulp-inject Nothing to inject into index.html.
[10:23:55] gulp-inject 10 files into index.html.
[10:23:55] Finished 'inject' after 82 ms
[10:23:55] Starting 'html'...
[10:23:56] gulp-inject 1 files into index.html.
[10:23:56] Finished 'fonts' after 2.02 s
[10:24:06] Finished 'other' after 12 s
[10:24:06] 'dist\' styles\vendor-389eb5336b.css 58.61 kB
[10:24:06] 'dist\' scripts\app-00a71b36aa.js 6.31 kB
[10:24:06] 'dist\' scripts\vendor-3552da7bef.js 407.58 kB
[10:24:06] 'dist\' index.html 584 B
[10:24:06] 'dist\' all files 473.08 kB
[10:24:06] Finished 'html' after 11 s
[10:24:06] Starting 'build'...
[10:24:06] Finished 'build' after 3.85 µs
```

**Comment by Swiip** [comment]
It's OS related, no problems with OSX or linux so it's only on Windows.

I think it's a problem of related paths from `gulp-inject` which doen's work with Stylus AND on Windows.

But I don't have a windows device to check and fix, maybe somedy else could check?

**Comment by Swiip** [comment]
I missed #601, can you checked than the correction does the trick?

**Comment by zckrs** [comment]
Confirm bug on Windows and #601 not fixed that

**Comment by zckrs** [comment]
Fixed by [v0.12.1](https://github.com/Swiip/generator-gulp-angular/releases/tag/v0.12.1)
